pkg/debugapi/postage.go

```diff
@@ -284,10 +332,23 @@ type chainStateResponse struct {
 func (s *Service) reserveStateHandler(w http.ResponseWriter, _ *http.Request) {
 	state := s.batchStore.GetReserveState()
 
+	commitment := int64(0)
+	if err := s.batchStore.Iterate(func(b *postage.Batch) (bool, error) {
+		commitment += int64(math.Pow(2.0, float64(b.Depth)))
```

Comment:
Does this iteration only happen on valid batches?... do we need to check if the `(normalised balance > cumulative payout)`?

Reply:
yes and no we don't... the batchstore should take care that all expired batches get _removed_ from the batchstore, therefore resulting in no iteration over those batches.